src/main/java/graphql/Directives.java

```diff
@@ -226,6 +247,23 @@ public class Directives {
             .definition(ONE_OF_DIRECTIVE_DEFINITION)
             .build();
 
+    @ExperimentalApi
+    public static final GraphQLDirective ErrorHandlingDirective = GraphQLDirective.newDirective()
+            .name(ERROR_HANDLING)
+            .description("This directive controls how to handle errors.")
+            .argument(newArgument()
+                    .name("onError")
+                    .type(nonNull(GraphQLEnumType.newEnum()
+                            .name(Enums.ON_ERROR)
+                            .value(Enums.ON_ERROR_PROPAGATE)
+                            .value(Enums.ON_ERROR_NULL)
+                            .build()))
+                    .defaultValueProgrammatic(Enums.ON_ERROR_PROPAGATE)
+                    .description("The URL that specifies the behaviour of this scalar."))
```

Comment:
I think this description is a copy paste from `specifiedBy`, could you update this?

Reply:
Thanks for catching this 🙏 . Fixed [there](https://github.com/graphql-java/graphql-java/pull/3772/commits/41863ea4ca974952b77e8d9b7a41c5b4f1703510#diff-cd15824ddc98b9f759a3469c0ca4df14a54820a0512d62954c8bff8d26c114caR263)

Out of curiosity, what is the reason there is both `DirectiveDefinition` and `GraphQLDirective`? I get that `DirectiveDefinition` is the AST one. But what is the need for `GraphQLDirective` too? Feels like it duplicates most of `DirectiveDefinition`?

Reply:
If some one was to build a GraphqlSchema programmatically (you dont need to use SDL technically) then this is a runtime ready representation of the directive.  A DirectiveDefinition is an AST element, typically from parsing SDL text.   A GraphQLDirective is similar but represents the runtime version of the directive in a GraphqlSchema